packages/runtime/test/src/lib/parsing.ts

```diff
@@ -64,6 +66,30 @@ describe("Convert integer to big endian", () => {
   });
 });
 
+describe("Parse string and integer, with bytes", () => {
+  it("should be equal when string in bytes is passed to uint decoder", () => {
+    let initialString = "50";
+    let first = stringToBytes(initialString);
+    let second = decodeUint64(first, DecodingMode.BIGINT);
+    let third = uint64ToBigEndian(second);
+    let final = convertToString(third);
+
+    // These two are not exactly equal
+    // actual - '\u0000\u0000\u0000\u0000\u0000\u000050'
+    // expected - '50'
+    assert.notEqual(final, initialString);
```

Comment:
I think we should compare `Uint8Arrays` here, as in stack (both in algorand network & runtime), strings are processed in bytes